backend/packages/Upgrade/src/api/services/AWSService.ts

```diff
@@ -68,7 +68,7 @@ export class AWSService {
   ): Promise<PromiseResult<AWS.SES.SendEmailResponse, AWS.AWSError>> {
     const param = {
       Destination: {
-        ToAddresses: [toAddress],
+        ToAddresses: [toAddress,],
```

Comment:
Remove extra comma

Reply:
This is expected as per documentation. Without this we get `final @domain missing` error

Reply:
Javascript will remove the comma. I don't think it will make any difference.
Can you paste the link of documentation here.

Reply:
ToAddresses expect 1 or more email id, so we need a comma. Please refer this: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property

<img width="466" alt="Screenshot 2022-05-07 at 3 32 16 PM" src="https://user-images.githubusercontent.com/33730817/167249507-38b86b60-c563-4c09-916a-2ef39eb46066.png">